### DISCUSSION TITLE
Repeated Issue with Remapping in foundry.toml, with @ referenced imports

### DISCUSSION THREAD

**Discussion by Shane0xM** [Cyfrin/foundry-full-course-cu#1878] [General]
Hello all, 

I have been having issues with my remappings in foundry and would greatly appreciate insight on how to best go about resolving this. 

Below is an example of the error that appears to be referenced correctly in my remappings:

1. Remapping: In this case looking to remap openzeppelin-contracts import, which has been successfully imported on my terminal

`remappings = ["@openzeppelin/contracts=lib/openzeppelin-contracts/contracts"]`

2. Import Reference in .sol file: 

`import {ERC20Burnable, ERC20} from "@openzeppelin/contracts/token/ERC20/extensions/ERC20.sol";`

**This appears underlined in red with a "File not found. Searched the following locations: " error.

When I use this alternative, it works fine without error:

`import {ERC20Burnable, ERC20} from "lib/openzeppelin-contracts/contracts/token/ERC20/extensions/ERC20Burnable.sol";`

3. Screenshot of Directory Layout for Reference:

![image](https://github.com/Cyfrin/foundry-full-course-cu/assets/169729951/859c087e-3087-4919-8dd7-f5855e5a27cc)

Could this be a settings issue? I tried to turn off and re-enable all extensions, verified the path is correct, and cleared cache through `forge clean` with no luck.

**Comment by EngrPips** [comment]
Hello @ShaneOxM, It appears that `ERC20.SOL` is not inside the extensions folder. Or am I not seeing correctly? Try the below import code, and let's see if it will work. 

`import {ERC20} from "@openzeppelin/contracts/token/ERC20/ERC20.sol"`

Looking forward to hearing from you.

**Comment by Shane0xM** [comment]
Spot on with this. Thanks much for the quick catch -- all resolved now.
